common/app/views/fragments/header.scala.html

```diff
@@ -20,6 +20,13 @@
                         <span class="js-profile-info control__info" data-test-id="sign-in-name">sign in</span>
                     </a>
                     <div class="js-profile-popup"></div>
+                    <div class="js-popup-signed-out popup popup--signed-out u-h">
+                        <a class="popup--signed-out__link" href="@Configuration.id.url/signin" data-link-name="Signed Out Link"></a>
+                        You are signed out of the Guardian. <span class="popup--signed-out__highlight">Sign In</span>
+                        <button class="js-popup-toggle u-button-reset popup__toggle">
```

Comment:
We need tracking on the close link. 

Can we add "Signed out Link | Close"... We'll need to double-check this works on buttons too.